File: server/src/main/java/lighting/server/ServerApplication.java
package lighting.server;

import lighting.server.monitor.MonitorService;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.messaging.simp.SimpMessagingTemplate;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableScheduling
public class ServerApplication {


	public static void main(String[] args) {
		SpringApplication.run(ServerApplication.class, args);
		//MonitorService monitorService = new MonitorService(null);
	}
}

File: server/src/main/scala/lassi/server/ServerApplication.scala
package lassi.server

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.scheduling.annotation.EnableScheduling

// run with -Dserver.port=8081
object ServerApplication {
  def main(args: Array[String]): Unit = {
    val app: Array[Class[_]] = Array(classOf[ServerApplication])
    SpringApplication.run(app, args)
  }
}

@SpringBootApplication
@EnableScheduling
class ServerApplication {
}
